sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClient.cs

```diff
@@ -93,10 +93,10 @@ public ArmClient(TokenCredential credential, ArmClientOptions options = default)
             ClientOptions = options?.Clone() ?? new ArmClientOptions();
             Pipeline = ManagementPipelineBuilder.Build(Credential, BaseUri, options ?? ClientOptions);
 
+            _tenant = new TenantOperations(ClientOptions, Credential, BaseUri, Pipeline);
             DefaultSubscription = string.IsNullOrWhiteSpace(defaultSubscriptionId)
                 ? GetDefaultSubscription()
                 : GetSubscriptions().TryGet(defaultSubscriptionId);
```

Comment:
`_tenant` is now used when creating susbcriptionContainer, so move the initialization forward.